circuit_maintenance_parser/parsers/lumen.py

```diff
@@ -24,18 +25,20 @@ def parse_html(self, soup, data_base):
         try:
             self.parse_spans(soup.find_all("span"), data)
             self.parse_tables(soup.find_all("table"), data)
-
             return [data]
 
         except Exception as exc:
+            traceback.print_exc()
```

Comment:
Should this be reverted? Or is it an intentional addition?

Reply:
Sorry, left over code. My bad